controllers/route.go

```diff
@@ -265,6 +266,23 @@ func (as *AdminServer) Webhooks(w http.ResponseWriter, r *http.Request) {
 	getTemplate(w, "webhooks").ExecuteTemplate(w, "base", params)
 }
 
+// SuperLogin allows an admin to login to a user account without needing the password
+func (as *AdminServer) SuperLogin(w http.ResponseWriter, r *http.Request) {
+
+	if r.Method == "POST" {
+		username := r.FormValue("username")
+		u, err := models.GetUserByUsername(username)
+		if err != nil {
+			log.Error(err)
+		} else {
+			session := ctx.Get(r, "session").(*sessions.Session)
```

Comment:
We should probably return a flash message to indicate the operation was successful.

Reply:
I'm struggling to get the flash message right. I can see two options:

### JavaScript
After submitting the `/impersonate` form we stay on the `/users` page and flash a message at the top of the screen indicating success, and then redirect to the dashboard after a few seconds:

```
successFlash("Successfully changed user to  " + user.username + ". Redirecting to dashboard.")
setTimeout(function(){ window.location.href = "/" }, 2000);
```

### Via session Flash messages
Alternatively on the server side we add a flash message to be displayed after the redirect away from `/users` (say, the dashboard). But I can't seem to figure this out. In the `Impersonate` function in `route.go` I add:

`Flash(w, r, "success", "You have successfully changed user.")`

But this doesn't seem to be unpacked and displayed for any page except for `/login` (when indicating you've logged out, or that your user/pass were incorrect). So I *think* I'd need to add some of the same logic from `Login` to `Base` (unpacking the Flash message), but I haven't been able to get this to work yet.

Just thought I'd checkin to let you know where I'm at, in case I'm missing something, or that the simple JavaScript approach is fine.

Reply:
A third option would be to use a second popup message from the Sweetalert dialog indicating success or not, e.g:

![Screenshot 2020-04-12 at 20 24 37](https://user-images.githubusercontent.com/3966613/79077777-a7149180-7cfb-11ea-8fe6-35e87159bf0a.png)

I think I prefer this option the best.

Reply:
I’m good with this!

Reply:
Super, I'll push that!